.github/workflows/pre-submit.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
       run: make manifests bundle-k8s bundle-reset test
```

Comment:
`bundle-k8s`?
If we are testing upstream OCP then it should be `bundle-metrics-ocp`, otherwise  the ` Test container build OCP Prometheus` should be changed to ` Test container build Kubernetes` and the target below as well

Reply:
This is for unittests and catching unwanted bundle changes (see ea5e7be), so it looks right to me. I'll make it clearer changing the name.

Regarding the "Test container build", while it is true we are more keen on k8s here, I think it would be better to test both, as it would be relevant to catch breaking changes at this early stage. For this reason I'd test both k8s _and_ ocp container builds

wdyt?

Reply:
SGTM